src/Compilers/Core/RebuildTest/CSharpRebuildTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+using BuildValidator;
+using Castle.Core.Logging;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Emit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Rebuild.UnitTests
+{
+    //[CompilerTrait(CompilerFeature.)]
+    public class CSharpRebuildTests : CSharpTestBase
+    {
+        [Theory]
+        [InlineData(Platform.Arm64)]
```

Comment:
Consider an approach similar to the one outlined in this blog for how to exhaustively test all of the `enum` members

https://willdot.net/27DotNetCoreEnumUnitTests/

Reply:
Fixed